src/tests/Interop/COM/NETServer/DispatchCoerceTesting.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Runtime.InteropServices;
+using Server.Contract;
+
+[ComVisible(true)]
+[Guid(Server.Contract.Guids.DispatchCoerceTesting)]
+public class DispatchCoerceTesting : Server.Contract.IDispatchCoerceTesting
+{
+    public int ReturnToManaged(short vt)
+    {
+        return 42;
```

Comment:
What does this mean? We don't use magic numbers for this domain. Please pass in what is desired to return.

Reply:
The managed server version shouldn't be called because it's not meaningful. Maybe a NotImplementedException is better?

Reply:
> Maybe a NotImplementedException is better?

+1